WORKING_GROUPS.md

```diff
@@ -129,8 +129,24 @@ The [OpenJS Code of Conduct][] applies to this Working Group.
 
 ## Current Working Groups
 
+* [Code of Conduct](#code-of-conduct)
 * [Standards](#standards)
 
+### [Code of Conduct](https://github.com/openjs-foundation/code-of-conduct/)
+
+The purpose of the Code of Conduct Working Group is to define and maintain a set of resources/processes for use by the Foundation, its teams and its projects.
+
+Responsibilities include:
+
+* Define and maintain overarching Code of Conduct and moderation processes, documentation and resources as they pertain to the Foundation and the Cross Project Council.
+* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined in https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md.
```

Comment:
```suggestion
* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined [here](https://github.com/openjs-foundation/cross-project-council/blob/master/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
```

Reply:
Perhaps instead of "here" it could say `defined in the [Foundation Code of Conduct Requirements]...`

Reply:
Updated. Thanks!